gnd/src/main/java/com/google/android/gnd/persistence/local/room/dao/FeatureDao.java

```diff
@@ -19,15 +19,16 @@
 import androidx.room.Dao;
 import androidx.room.Query;
 import com.google.android.gnd.persistence.local.room.entity.FeatureEntity;
+import com.google.android.gnd.persistence.local.room.models.EntityState;
 import io.reactivex.Flowable;
 import io.reactivex.Maybe;
 import java.util.List;
 
 /** Provides low-level read/write operations of {@link FeatureEntity} to/from the local db. */
 @Dao
 public interface FeatureDao extends BaseDao<FeatureEntity> {
-  @Query("SELECT * FROM feature WHERE project_id = :projectId")
-  Flowable<List<FeatureEntity>> findByProjectIdStream(String projectId);
+  @Query("SELECT * FROM feature WHERE project_id = :projectId AND state = :state")
+  Flowable<List<FeatureEntity>> findByProjectIdStream(String projectId, EntityState state);
```

Comment:
Using our convention in the Wiki, should should be called `findOnceAndStream` (or `findByProjectIdAndEntityStateOnceAndStream`, but that seems excessive).